src/Api/Workflows.php

```diff
@@ -101,12 +109,13 @@ public function enrollmentsForContact($contact_id)
     /**
      * Get past events for contact from a workflow.
      *
-     * @param int $workflow_id
-     * @param int $contact_id
-     * @param array $params Optional parameters.
+     * @param int   $workflow_id Wofkflow Id
+     * @param int   $contact_id  Contact Id
+     * @param array $params      Optional parameters.
+     *
      * @return mixed
      */
-    public function pastEventsForContact($workflow_id, $contact_id, $params)
+    public function pastEventsForContact($workflow_id, $contact_id, array $params)
```

Comment:
I don't like type-hinting for array everywhere because that stops people from using objects that implement `ArrayAccess` and `Iterator` and type-hinting for array breaks that.

For example people can pass in a [Query](https://github.com/ryanwinchester/hubspot-php/blob/master/src/Http/Query.php) object instead of an array here, too.